<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Botonera</class>
 <widget class="QMainWindow" name="Botonera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>821</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="barra_estado">
       <item>
        <widget class="QLabel" name="kerberos">
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_kerberos">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Renovar</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_DB">
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_reconectaDB">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Conecta DB</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_username">
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_ip">
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNuevo">
    <property name="title">
     <string>Archi&amp;vo</string>
    </property>
    <addaction name="actionNuevo_Men"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAplicaciones">
    <property name="title">
     <string>Ap&amp;licaciones</string>
    </property>
    <addaction name="actionUsuarios"/>
    <addaction name="actionSoporte"/>
    <addaction name="actionCronos"/>
    <addaction name="actionWebmail"/>
    <addaction name="actionBeiro"/>
    <addaction name="actionGlpi"/>
    <addaction name="actionOcs"/>
    <addaction name="actionActiveDirectory"/>
    <addaction name="actionISL"/>
    <addaction name="actionConfigurar"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuNuevo"/>
   <addaction name="menuAplicaciones"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUsuarios"/>
   <addaction name="actionSoporte"/>
   <addaction name="actionSedes"/>
   <addaction name="actionCronos"/>
   <addaction name="actionWebmail"/>
   <addaction name="actionBeiro"/>
   <addaction name="actionGlpi"/>
   <addaction name="actionOcs"/>
   <addaction name="actionActiveDirectory"/>
   <addaction name="actionISL"/>
   <addaction name="actionAtalaya"/>
   <addaction name="actionConfigurar"/>
   <addaction name="actionSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionUsuarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/usuarios.png</normaloff>../../../usr/share/grx/asistencia/iconos/usuarios.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Usuarios</string>
   </property>
   <property name="toolTip">
    <string>Usuarios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCronos">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/cronos.png</normaloff>../../../usr/share/grx/asistencia/iconos/cronos.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cronos</string>
   </property>
   <property name="toolTip">
    <string>Cronos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionWebmail">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/WebMail.png</normaloff>../../../usr/share/grx/asistencia/iconos/WebMail.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Webmail</string>
   </property>
   <property name="toolTip">
    <string>Webmail</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F3</string>
   </property>
  </action>
  <action name="actionBeiro">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/beiro_cartera.png</normaloff>../../../usr/share/grx/asistencia/iconos/beiro_cartera.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Beiro</string>
   </property>
   <property name="toolTip">
    <string>Beiro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionGlpi">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/glpi.png</normaloff>../../../usr/share/grx/asistencia/iconos/glpi.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Glpi</string>
   </property>
   <property name="toolTip">
    <string>Glpi</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionOcs">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/ocs.png</normaloff>../../../usr/share/grx/asistencia/iconos/ocs.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ocs</string>
   </property>
   <property name="toolTip">
    <string>Ocs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
  </action>
  <action name="actionActiveDirectory">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/AD.png</normaloff>../../../usr/share/grx/asistencia/iconos/AD.png</iconset>
   </property>
   <property name="text">
    <string>&amp;activeDirectory</string>
   </property>
   <property name="toolTip">
    <string>Active Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F7</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/salida.jpg</normaloff>../../../usr/share/grx/asistencia/iconos/salida.jpg</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F8</string>
   </property>
  </action>
  <action name="actionNuevo_Men">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/logo.png</normaloff>../../../usr/share/grx/asistencia/iconos/logo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nueva Botonera</string>
   </property>
  </action>
  <action name="actionISL">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/ISL50x50.png</normaloff>../../../usr/share/grx/asistencia/iconos/ISL50x50.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ISL</string>
   </property>
   <property name="toolTip">
    <string>ISLlight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionConfigurar">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/update.png</normaloff>../../../usr/share/grx/asistencia/iconos/update.png</iconset>
   </property>
   <property name="text">
    <string>Co&amp;nfigurar</string>
   </property>
  </action>
  <action name="actionSoporte">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/equipos.gif</normaloff>../../../usr/share/grx/asistencia/iconos/equipos.gif</iconset>
   </property>
   <property name="text">
    <string>&amp;Soporte</string>
   </property>
   <property name="toolTip">
    <string>Soporte</string>
   </property>
  </action>
  <action name="actionSedes">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/sedes.png</normaloff>../../../usr/share/grx/asistencia/iconos/sedes.png</iconset>
   </property>
   <property name="text">
    <string>Sedes</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/300px-Granada_-_Mapa_municipal.svg.png</normaloff>../../../usr/share/grx/asistencia/iconos/300px-Granada_-_Mapa_municipal.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de ...</string>
   </property>
  </action>
  <action name="actionAtalaya">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/grx/asistencia/iconos/atalaya.png</normaloff>../../../usr/share/grx/asistencia/iconos/atalaya.png</iconset>
   </property>
   <property name="text">
    <string>Atalaya</string>
   </property>
   <property name="toolTip">
    <string>Atalaya</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
